configure.ac

```diff
@@ -632,6 +632,11 @@ case "$enable_unix" in
   *) AC_MSG_ERROR([bad value $enable_unix for --enable-unix option]) ;;
 esac
 
+AC_ARG_WITH([misc-conv-bufsize],
+AS_HELP_STRING([--with-misc-conv-bufsize=<number>],[Size of input buffer for libpam_misc's misc_conv() conversation function, default=4096]),
+[], [with_misc_conv_bufsize=4096])
```

Comment:
The line is too long, please wrap and indent.

Reply:
Done!